changelog.d/pa-2694.added

```diff
@@ -0,0 +1 @@
+Scala: Can now parse top-level definitions (as added in Scala 3)
```

Comment:
How many PRs this PR contains?
I still don't understand how all your PR contain stuff from other PRs. How do you use git to make your PR? Or it is because each PR depends on one another and you get confused in using gits with so many branches?